src/context/WalletProvider/KeepKey/components/RecoverySentenceInvalid.tsx

```diff
@@ -0,0 +1,38 @@
+import { WarningTwoIcon } from '@chakra-ui/icons'
+import { Button, ModalBody, ModalHeader } from '@chakra-ui/react'
+import { useState } from 'react'
+import { Text } from 'components/Text'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+import { useKeepKeyRecover } from '../hooks/useKeepKeyRecover'
+
+export const KeepKeyRecoverySentenceInvalid = () => {
+  const [loading, setLoading] = useState(false)
+  const {
+    state: { wallet },
+  } = useWallet()
+  const sendKeepKeyRecover = useKeepKeyRecover()
+
+  const handleRetryPress = async () => {
```

Comment:
`press` would imply pressing a button on the KK itself
```suggestion
  const handleRetryClick = async () => {
```